.pre-commit-config.yaml

```diff
@@ -1,7 +1,8 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 
-files: "(nilearn/de.*)|(.*binder/)|(.*circleci/)|(.*github/)|(maint_tools/.*)|(setup.*)|(.*coverage)|(.*toml$)|(.*in$)|(.*yml)|(.*yaml)|(CONTRIBUTING.rst)|(AUTHORS.rst)|(LICENSE)|(Makefile)|(.*gitignore)"
+files: "(nilearn/de.*)|(.*binder/)|(.*circleci/)|(.*github/)|(maint_tools/.*)|(setup.*)|(.*coverage)|(.*toml$)|(.*in$)|(.*yml)|(.*yaml)|(CONTRIBUTING.rst)|(AUTHORS.rst)|(LICENSE)|(Makefile)|(.*gitignore)|(doc/.*)"
+exclude: "^(doc/whats_new.rst)|(doc/changes/(names|latest|0\\.9\\.[0-2]|0\\.10\\.0).rst)|(doc/(glm/first_level_model\\.rst|introduction\\.rst|maintenance\\.rst|manipulating_images/input_output\\.rst|modules/datasets\\.rst|modules/reference\\.rst|references\\.bib|themes/nilearn/static/nature\\.css_t))$"
```

Comment:
Will probably lead to some merge conflicts with the PR for reformatting of the example, will handle those in due time.